api/types/types.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/docker/docker/api/types/image"
 	"github.com/docker/docker/api/types/swarm"
 	"github.com/docker/docker/api/types/volume"
+	"github.com/docker/docker/daemon/capabilities"
```

Comment:
Wondering if we can reverse this; basically, we want to avoid anything inside `daemon/` to be imported outside of our repository, so perhaps we need (at least the "types" to be defined as a package in `api/xxx`.

Reply:
Think so! Initially I had the capabilities logic in `daemon/`, so that wouldn't work because `daemon/` would have to import `api/` and that would cause issues because `api/` imports `daemon` somewhere down the chain, so I changed it. However, now that we have a capabilities package (which doesn't import the `daemon`) we can have it import `api`.